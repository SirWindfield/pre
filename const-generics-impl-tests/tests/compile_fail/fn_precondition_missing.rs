include!("../../../tests/fn_precondition_missing.rs");
