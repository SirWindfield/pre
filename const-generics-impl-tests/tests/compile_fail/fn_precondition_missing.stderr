error[E0308]: mismatched types
  --> $DIR/fn_precondition_missing.rs:12:9
   |
12 |         #[assert_precondition(holds(valid_ptr(ptr), reason = "`ptr` comes from a reference"))]
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected a tuple with 2 elements, found one with 1 element
   |
   = note: expected struct `std::marker::PhantomData<(pre::const_generics_types::ValidPtrConditionHolds<"ptr">, pre::const_generics_types::CustomConditionHolds<"must point to `42`">)>`
              found struct `std::marker::PhantomData<(pre::const_generics_types::ValidPtrConditionHolds<"ptr">,)>`
   = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)
